packages/react-core/src/components/Alert/Alert.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { StyleSheet, css, getModifier } from '@patternfly/react-styles';
+import PropTypes from 'prop-types';
+import styles from '@patternfly/patternfly-next/components/Alert/styles.css';
+
+import AlertIcon from './AlertIcon';
+import AlertBody from './AlertBody';
+import AlertAction from './AlertAction';
+import { capitalize } from '../../internal/util';
+
+const utils = {
+  ...StyleSheet.create({
+    srOnly: {
+      position: 'absolute',
+      width: '1px',
+      height: '1px',
+      padding: '0',
+      margin: '-1px',
+      overflow: 'hidden',
+      clip: 'rect(0,0,0,0)',
+      border: '0'
+    }
+  })
+};
+
+export const alertVariant = {
```

Comment:
This does not match the casing of the `.d.ts`.  Make this `AlertVariant` to match button and the `d.ts`.